README.md

```diff
@@ -146,7 +146,7 @@ For automated tests of the complete example using [bats](https://github.com/bats
       attributes                = var.attributes
       tags                      = var.tags
       subnet_ids                = module.subnets.public_subnet_ids
-      instance_types            = var.instance_types
+      instance_type             = var.instance_type
```

Comment:
So I really struggled with this one..

I don't like the impact this change has to existing folks, but on the other hand I also don't like misleading the user into thinking that we can accept multiple instance types when it's unsupported. 

The gist of it is, now that we're defining our own launch template, we have to specify the instance type in `aws_launch_template` instead of `aws_eks_node_group`. And well, the `aws_launch_template` resource accepts a string (i.e. only one instance type). 

In previous versions, we were passing the `instance_types` into `aws_eks_node_group` as a list, although the documentation states that the EKS API "only accepts a single value in the set."

Anyhow, it sounds like multiple instance types may be supported at some point in the future, but for now it seems prudent to limit the user to a single type. I'm open to further input if anyone has any other ideas -- it'd be easy enough to revert/change if needed anyhow.

Reply:
Yup, totally noticed that multiple instance types weren't supported when I tried providing multiple originally here